video/out/vulkan/context_wayland.c

```diff
@@ -99,6 +101,11 @@ static bool wayland_vk_init(struct ra_ctx *ctx)
     if (!ra_vk_ctx_init(ctx, vk, params, VK_PRESENT_MODE_MAILBOX_KHR))
         goto error;
 
+    // Assume user is using vk_hdr_layer for color management.
+    const char *hdr_wsi = getenv("ENABLE_HDR_WSI");
```

Comment:
We could check if the layer is actually loaded too, but it's fine for now I guess.

Reply:
Is there an easy way to do that from our end?

Reply:
Okay should work now.